### PR TITLE
Added New Spam Box

### PR DESCRIPTION
**Pull request by defnax**
* Added New Spam Box

### CODE REVIEW

File: retroshare-gui/src/gui/msgs/MessagesDialog.cpp [hunk 1]
```diff
@@ -432,6 +437,16 @@ void MessagesDialog::fillQuickView()
 		itemToSelect = item;
 	}
 
+	item = new QListWidgetItem(tr("Spam"), ui.quickViewWidget);
+	item->setIcon(QIcon(IMAGE_SPAM_ON));
```
Comment (csoler) [inline retroshare-gui/src/gui/msgs/MessagesDialog.cpp:441]: use FileDefs here!

**Comment by csoler** [comment]
Almost ready. Just missing the FilesDefs I mentioned above. Thx!

**Comment by defnax** [comment]
ok i can do it

File: retroshare-gui/src/gui/msgs/MessagesDialog.cpp (hunk 1 above)
Comment (csoler) [reply retroshare-gui/src/gui/msgs/MessagesDialog.cpp:441]: I'm realizing that these items will only be used once actually
so it's not that critical
it's only useful if the same icons are used somewhere else too
I originally thought that you where creating items in a loop, but no

**Pull request merged**